### ISSUE TITLE
Element inspector: screenreader cannot distinguish buttons

### ISSUE THREAD

**Issue by SteveMarvin** [performant-software/faircopy#358]
![image](https://user-images.githubusercontent.com/18423533/146037222-3e36e13b-25c9-4ad2-a5d8-9b0ca78037b1.png)

Screenreader only says "button" for both the check and the X
